src/System.Data.SqlClient/tests/ManualTests/SQL/ConnectivityTests/ConnectivityTest.cs

```diff
@@ -169,5 +179,35 @@ public void SqlConnectionOpen()
                 _doneEvent.Set();
             }
         }
+
+        [CheckConnStrSetupFact]
+        public static void ConnectionKilledTest()
+        {
+            try
+            {
+                // Setup Database and Table.
+                DataTestUtility.RunNonQuery(s_connectionString, s_createDatabaseCmd);
+                DataTestUtility.RunNonQuery(s_dbConnectionString, s_createTableCmd);
+
+                // Kill all the connections and set Database to SINGLE_USER Mode.
+                DataTestUtility.RunNonQuery(s_connectionString, s_alterDatabaseSingleCmd);
```

Comment:
This test has the potential to disrupt other tests, if run in parallel. 
There are two things that can be done, use the RemoteExecutor to spin up a new process to run this test.
Alternatively, instead of using the Single User and Multiuser transition, Open a connection, kill the SPID from the SQL Server using another connection and try to get a connection from the first pool.

Reply:
https://github.com/dotnet/corefx/pull/31631/files#diff-22859f3a645ce5f4ae643d8c29205840R22
The connections are being severed on the dynamically generated Database Name, therefore I don't believe the test needs any changes, it should work in parallel as is.